composer.json

```diff
@@ -19,8 +19,8 @@
         "php": ">=5.6.0",
         "behat/behat": "~3.0",
         "atoum/atoum": "~1.0|~2.0|~3.0|3.0.x-dev",
-        "symfony/property-access": "~2.4|~3.0",
-        "justinrainbow/json-schema": ">=3.0 <=6.0",
+        "symfony/property-access": "~2.5|~3.0",
```

Comment:
2.4 release of `symfony/property-access` leads to bugs in features about json inspector evaluation of json path containing the `-` character.